mod-invoice-storage/schemas/invoice.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description": "invoice",
+  "javaName": "Invoice",
```

Comment:
Is it needed to properly generate enums?

Reply:
Could not find a way around enums. This helped to not generate additional InvoiceSnapshot for audit event attribute and reuse existing Invoice class